src/snakemake/path_modifier.py

```
@@ -4,6 +4,7 @@
 __license__ = "MIT"
 
 import os
+from typing import Callable, List
```

Ruff (0.8.2)

`typing.Callable` imported but unused

Remove unused import

(F401)